wiki/windows-feature-store.md

```diff
@@ -32,35 +32,35 @@ Remember, not all feature IDs will have any noticeable change, or any change at
-| 38613007   | New Details Panel in File Explorer   | 25276           |
-| 41040327   | File Explorer "Gallery" Entry        | 25272*          |
-| 42295138 + 41040327 | XAMLFolderViewSupport       | 25276           |
-| 41162753   | WlanLocationPermission               | 25281           |
-| 39880030 + 40522394 + 41744267 + 41861575 | New Desktop Spotlight UI treatments | 25281
-| 40268500   | New Desktop Spotlight UI treatments (ID's above are required for this) | 25281 | Yes 0-3 |
-| 29785186   | Taskbar never combine                | 23466           |
-| 23214784 + 41967741 | Turn off toast banner notification suggestions | 23466 |
-| 43572692   | New taskbar search-on-hover experience | 23466         |
-| 35262205   | Dynamic lighting settings              | 23466         |
-| 40950262   | Modern File Explorer navigation bar    | 23466         |
-| 40213648   | Full Emoji 15.0 support                | 23466         |
+| Feature ID                                | Description                                                            | Minimum Version       | Variant(s) |
+| ----------------------------------------- | ---------------------------------------------------------------------- | --------------------- | ---------- |
+| 36302090                                  | Modern "Open with" dialog                                              | 22622.290             |
+| 37634385                                  | Tabs in File Explorer                                                  | 21536                 |
+| 36354489                                  | Updated navigation pane                                                | 21536                 |
+| 39145991                                  | Enable to use above features on Beta                                   | 22621.160             |
+| 37010913                                  | Move search button to left                                             | 21536                 |
+| 37969115                                  | Show the desktop search bar                                            | 25120                 |
+| 40887771                                  | Move new search box next to widgets                                    | 25217                 |
+| 39072097                                  | New taskbar loading animation                                          | 25179 / 22622.575     |
+| 39072097                                  | Task Manager on Taskbar Conext Menu                                    | 25211                 |
+| 38764045                                  | New system tray                                                        | 25211                 |
+| 34878152                                  | Animated Settings navigation panel icons                               | 25197                 |
+| 26008830                                  | Tablet-optimized Taskbar                                               | 25197                 |
```

Comment:
A previous PR changed this from Taskbar to taskbar and now yours changes it back - can we get someone to check which is correct? >_>

Reply:
Probably accidental thing as a result of fixing the merge conflict. Will work on it.

Reply:
@Erisa reminds me; i changed it from "Taskbar" to "taskbar" was that wrong? kinda didn't make sense to me that the t was capitalized

Reply:
@Erisa Which one are you referring to exactly now? Just realized and I'm confused.